cmd/oras/tag/tag.go

```diff
@@ -70,5 +82,22 @@ func tagManifest(opts tagOptions) error {
 		return errors.NewErrInvalidReference(repo.Reference)
 	}
 
-	return oras.Tag(ctx, repo, opts.srcRef, opts.targetRef)
+	rp := &wrapper{
+		repo,
+	}
+
+	return oras.TagN(ctx, rp, opts.srcRef, opts.targetRef, opts.defaultTagNOptions)
+}
+
+type wrapper struct {
+	*remote.Repository
+}
+
+// Override PushReference method to print off which tag(s) were added successfully.
+func (w *wrapper) PushReference(ctx context.Context, expected ocispec.Descriptor, content io.Reader, reference string) error {
+	if err := w.Repository.PushReference(ctx, expected, content, reference); err != nil {
+		return fmt.Errorf("failed %s", reference)
+	}
+	fmt.Println("tagged", reference)
```

Comment:
Printing using `fmt.Println` is not go-routine safe. Please consider https://github.com/oras-project/oras/blob/main/cmd/oras/internal/display/print.go#L45

Reply:
using `display.Print` now